composer.json

```diff
@@ -24,6 +24,7 @@
         "php": ">=5.4.8",
         "symfony/event-dispatcher": "~2.3",
         "symfony/options-resolver": "~2.3",
+        "symfony/filesystem": "~2.3",
```

Comment:
i would prefer to move it to require-dev and document you need this if you want to use the symfony CacheKernel based tagging, yes. and have a class_exists check with the same information in the place where we would run into the problem if the user did not read the doc...